crates/ruff_python_ast/src/comparable.rs

```diff
@@ -1196,16 +1204,19 @@ impl<'a> From<&'a ast::TypeParam> for ComparableTypeParam<'a> {
 pub struct TypeParamTypeVar<'a> {
     pub name: &'a str,
     pub bound: Option<Box<ComparableExpr<'a>>>,
+    pub default_value: Option<Box<ComparableExpr<'a>>>,
```

Comment:
Same suggestion as in the AST node, we should rename this to `default` for consistency.

Reply:
Mentioned this above, will rename it everywhere if you confirm that you prefer this name.